templates/zerver/base.html

```diff
@@ -16,8 +16,33 @@
                 {% endif %}
             {% endif %}
         {% endblock %}
+        <!-- The order of the alternate_icon and the icon here is
+        important.
+
+        The spec (https://html.spec.whatwg.org/multipage/links.html#rel-icon)
+        mentions "If there are multiple equally appropriate icons,
+        user agents must use the last one declared in tree order at
+        the time that the user agent collected the list of icons."
+
+        If the order was reversed and the user had
+        realm_icon_as_favicon set to true, we would have a bug
+        whereby after once getting to 0 unreads (ie this would not be
+        seen on first page load) the png image would be favoured in
+        place of the svg image ie the count would not be shown.
+
+        It's unclear why the same bug isn't seen with the default
+        zulip svg.
+
+        Also, the above bug may be chrome specific, (see:
+        https://crbug.com/1140920)
+        -->
+        {% if user_profile and user_profile.realm_icon_as_favicon %}
+        <link id="alternate_icon" rel="alternate icon" href="{{ page_params.realm_icon_url }}" sizes="any"/>
+        <link id="favicon" rel="icon" href="{{ page_params.realm_icon_url }}" sizes="any"/>
```

Comment:
Why do we need `sizes="any"`, and why only in this case?

Reply:
To the best of my understanding, adding `sizes="any"` is what makes the browser (or at least, chrome) consider these to be "equally appropriate icons". I believe this because without this attribute, we see the bug described above.

In addition, I believe we do not need this attribute with the zulip logo because the svg is already considered more appropriate than the png (which may be because they both have the same dimensions?).